<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePropertiesDialog</class>
 <widget class="QDialog" name="TemplatePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="toolTip">
        <string>Enter the name of the template/group. Templates are autocompleted upon this name.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="toolTip">
        <string>Enter a description for the template</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="groupCombo">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="templateLabel">
       <property name="text">
        <string>Template:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="3">
      <widget class="QTextEdit" name="templateEdit">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Enter the text of the template</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Template Text&lt;/b&gt;
&lt;p&gt;Enter the template text in this area. Every occurrence of $VAR$ will be replaced
by the associated text when the template is applied.  Predefined variables may be used in the template. The separator character might
be changed via the preferences dialog.&lt;/p&gt;
&lt;p&gt;Press the help button for more information.&lt;/p&gt;</string>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>84</width>
         <height>98</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>groupCombo</tabstop>
  <tabstop>templateEdit</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemplatePropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemplatePropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
